RPS.html

```diff
@@ -1,158 +1,96 @@
+        return draw;
   
-  } else {
-    console.log("Something goes wrong...");
-  }
-}
-
-/*function capitalize(string) {
-  return string.charAt(0).toUpperCase() + string.slice(1).toLowerCase();
-}*/
-
-function rockGame() {
-  let playerSelection = "Rock";
-  const computerSelection = computerPlay();
-
-  const resultDiv = document.querySelector('.result');
-  const p1 = document.createElement('p');
-  const p2 = document.createElement('p');
-  const p3 = document.createElement('p');
-
-  p1.textContent = playRound(playerSelection, computerSelection);
-  p2.textContent = "User selects: " + playerSelection;
-  p3.textContent = "Comp selects: " + computerSelection;
-
-  resultDiv.appendChild(p1);
-  resultDiv.appendChild(p2);
-  resultDiv.appendChild(p3);
-}
-
-function paperGame() {
-  let playerSelection = "Paper";
-  const computerSelection = computerPlay();
-  
-  const resultDiv = document.querySelector('.result');
-  const p1 = document.createElement('p');
-  const p2 = document.createElement('p');
-  const p3 = document.createElement('p');
-
-  p1.textContent = playRound(playerSelection, computerSelection);
-  p2.textContent = "User selects: " + playerSelection;
-  p3.textContent = "Comp selects: " + computerSelection;
-
-  resultDiv.appendChild(p1);
-  resultDiv.appendChild(p2);
-  resultDiv.appendChild(p3);
-}
-
-function scissorsGame() {
-  let playerSelection = "Scissors";
-  const computerSelection = computerPlay();
-  
-  const resultDiv = document.querySelector('.result');
-  const p1 = document.createElement('p');
-  const p2 = document.createElement('p');
-  const p3 = document.createElement('p');
-
-  p1.textContent = playRound(playerSelection, computerSelection);
-  p2.textContent = "User selects: " + playerSelection;
-  p3.textContent = "Comp selects: " + computerSelection;
-
-  resultDiv.appendChild(p1);
-  resultDiv.appendChild(p2);
-  resultDiv.appendChild(p3);
-}
-
-const rockBtn = document.querySelector('#rock');
-const paperBtn = document.querySelector('#paper');
-const scissorsBtn = document.querySelector('#scissors');
-
-rockBtn.addEventListener('click', rockGame);
-paperBtn.addEventListener('click', paperGame);
-scissorsBtn.addEventListener('click', scissorsGame);
-
- </script>
- 
-</body>
+      } else {
+        console.log("Something goes wrong...");
+      }
+    }
+
+    function game(playerSelection) {
+      const computerSelection = computerPlay();
+
+      const resultDiv = document.querySelector('.result');
+      const p1 = document.createElement('p');
+      const p2 = document.createElement('p');
+      const p3 = document.createElement('p');
+
+      p1.textContent = playRound(playerSelection, computerSelection);
+      p2.textContent = "User selects: " + playerSelection;
+      p3.textContent = "Comp selects: " + computerSelection;
+
+      resultDiv.appendChild(p1);
+      resultDiv.appendChild(p2);
+      resultDiv.appendChild(p3);
```

Comment:
You can turn it into a `forEach` just for fun. Create an array of `p1`, `p2` and `p3` and then you will only need one line of `appendChild`. Like so
```js
[p1, p2, p3].forEach((element) => resultDiv.appendChild(element));
```